sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -263,9 +263,7 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
         }
       }
 
-      val concurrentGpuTasks = conf.concurrentGpuTasks
-      logInfo(s"The number of concurrent GPU tasks allowed is $concurrentGpuTasks")
```

Comment:
It may be nice (but not required) to output this based on the initial value of the config at this stage. That way the executor log tells you what we started with. I wonder if this is useful in the case where users are not setting it (the executor log would contain this information).

Conversely, if people are setting it, then this log is meaningless, and I see why it was removed.